inst/htmljs/index.html

```diff
@@ -12,6 +12,8 @@
     <script type="text/javascript" src="vendor/jquery-1.11.3.min.js"></script>
     <script type="text/javascript" src="vendor/selectize.min.js"></script>
     <link rel="stylesheet" type="text/css" href="vendor/selectize.css" />
+    <script src="https://cdn.jsdelivr.net/npm/driver.js@1.0.1/dist/driver.js.iife.js"></script>
```

Comment:
can you please copy these to inst/htmljs/vendor? and then use relative links here? (as with jquery etc above)
this approach is more stable (all files hosted on same server, don't need to rely on external cdn)